src/components/accordion/accordion-header.tsx

```diff
@@ -0,0 +1,84 @@
+import styled, { css } from 'styled-components';
+import { TextStyles } from '../typography/typography-v2';
+import { ellipsis } from 'polished';
+import * as ReachAccordion from '@reach/accordion';
+import { theme } from '../layout/theme';
+import ChevronSVG from './chevron.svg';
+import { useAccordionItemContext } from '@reach/accordion';
+import React from 'react';
+
+interface AccordionHeaderProps {
+  children: React.ReactNode;
+}
+
+interface AccordionChevronProps {
+  open: boolean;
+  className?: string;
+}
+
+const AccordionTitleHeadline = styled.span`
+  ${TextStyles.headlineXS}
+  ${ellipsis()}
+  padding-right: 20px;
+  color: inherit;
+`;
+
+const resetButton = css`
+  padding: 0;
+  margin: 0;
+  appearance: none;
+  background: 0;
+  border: 0;
+  box-shadow: none;
+`;
```

Comment:
I had the feeling this makes the actual button styles more readable so I grouped that reset. Might be useful globally but for now local is good.